liquibase-core/src/main/resources/www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd

```diff
@@ -920,6 +920,7 @@
             <xsd:attribute name="tableName" type="xsd:string" use="required"/>
             <xsd:attribute name="columnName" type="xsd:string" use="required"/>
             <xsd:attribute name="remarks" type="xsd:string"/>
+            <xsd:attribute name="columnDataType" type="xsd:string"/>
```

Comment:
Since the 3.8 XSD is released and frozen, this will have to go into a new 3.9 xsd